generator/src/test/java/com/scottlogic/deg/generator/cucumber/After.feature

```diff
@@ -0,0 +1,323 @@
+#Generator expected to generate in ascending order where there are bracketing constraints
+Feature: User can specify that a temporal date is lower than, but not equal to, a specified threshold
+
+  Background:
+    Given the generation strategy is full
+    And there is a field foo
+    And foo is of type "temporal"
+
+#Related to #667 - Expected time increment is one millisecond
+@ignore
+Scenario: 'After' valid date is successful
+  Given foo is after 2018-09-01T00:00:00.000
+    And the generator can generate at most 4 rows
+  Then the following data should be generated:
+    | foo                     |
+    | 2018-09-01T00:00:00.001 |
+    | 2018-09-01T00:00:00.002 |
+    | 2018-09-01T00:00:00.003 |
+    | 2018-09-01T00:00:00.004 |
+
+Scenario Outline: 'After' invalid datetime fails with error
+  Given foo is after <dateValue>
+  Then I am presented with an error message
+    And no data is created
+  Examples:
+    | dateValue               |
+    | 2018-09-32T00:00:00.000 |
+    | 2018-09-01T25:00:00.000 |
+    | 2018-13-01T00:00:00.000 |
+
+#Issue #610 - Leap year date invalid
+@ignore
+Scenario: 'After' non-existent leap year date fails with error
+  Given foo is after 2019-02-29T00:00:00.000
+  Then I am presented with an error message
+  And no data is created
+
+  ### after ###
```

Comment:
aren't all these tests for "after", what is this comment implying?

Reply:
The triple hash comments are to show what after is combined with in the set (I copied this format from existing features)

Reply:
I'm not sure the existing format helped, I'd probably remove

Reply:
I think this marker helps to highlight which section the tests come from in the test matrix, hence they were included.
I'd keep.

For completeness the scenarios in this 'section' are where the `after` operand is combined with another `after` operand, therefore the comment highlights this combination.